sdk/metrics/src/test/java/io/opentelemetry/sdk/metrics/internal/aggregator/DoubleExponentialHistogramAggregatorTest.java

```diff
@@ -122,6 +135,45 @@ void testInvalidRecording() {
     assertThat(acc.getZeroCount()).isEqualTo(0);
   }
 
+  @ParameterizedTest
```

Comment:
Tested this for both `mapCounter` and `circularBufferCounter` impls because they both have index logic that can possibly overflow.